tests/modules/group_config_test/group_config_test.info.yml

```diff
@@ -2,4 +2,4 @@ name: 'Group config test'
 description: 'Configuration import test module for the localgov_menu_link_group config entity.'
 type: module
 package: Testing
-core_version_requirement: ^8.8 || ^9
+core_version_requirement: ^9 || ^10
```

Comment:
Should we remove 8 support here?
Has it broken D8 support?
If not, maybe no harm to leave it in place for those who may be still using it on Drupal 8.
Otherwise I think we need at least a minor version and maybe even a core requirement bump if dropping support for Drupal 8.

Reply:
We don't support Drupal 8 any longer. Any D8 sites out there are now insecure and I suspect not being updated anyway. I have been removing D8 support from all D10 branches. For example this got merged recently https://github.com/localgovdrupal/localgov_topics/blob/4302899721f032f257d4ee7c40b3695ee6852a24/localgov_topics.info.yml#L5

I suggest it's a tech governance thing to decide policy if there's a disagreement here.

I'm suggesting minor version bumps for D10 support, although that didn't happen for Topics.

Reply:
This is actully only the test module thats included so I don't think it actully matters.